src/dso_api/dynamic_api/filterset.py

```diff
@@ -5,12 +5,30 @@
 from typing import Type
 
 from django.contrib.gis.db.models import GeometryField
+from django.db import models
 
 from dso_api.dynamic_api.utils import snake_to_camel_case
 from dso_api.lib.schematools.models import DynamicModel
 from rest_framework_dso.filters import DSOFilterSet
 
 
+# These extra lookups are available for specific data types:
+_comparison_lookups = ["exact", "gte", "gt", "lt", "lte"]
+_identifier_lookups = ["exact", "in"]
+DEFAULT_LOOKUPS_BY_TYPE = {
+    models.AutoField: _identifier_lookups,
+    models.IntegerField: _comparison_lookups,
```

Comment:
Bij integer fields zou ook 'in' mogelijk moeten zijn. Als integer als enumeration  wordt gebruikt oid. of als het toch naar  andere table verwijst, maar niet als FK gemodelleerd kan worden.

Reply:
Goede, voeg ik toe, ook voor float/decimals.